### ISSUE TITLE
My model appearing on other players

### ISSUE THREAD

**Issue by SyntaxError73** [tom5454/CustomPlayerModels#555]
I am playing on a multiplayer server with this mod, and depending on the camera angle of the other player, it'll "blink" my model over his skin. He will switch back and forth from his skin to my model. It only happens on his end so he recorded the issue for me.

It looks fine on my end, but this is what he sees:

https://github.com/tom5454/CustomPlayerModels/assets/36898706/eab0877a-8d86-4d8c-bd25-fa3c491cf8e9

I am running the mod on minecraft 1.20.1, Forge 47.2.0

**Comment by SyntaxError73** [comment]
NEW INFORMATION:

Based on the video provided, my model "transfers" from my player to his whenever I am on his screen, when I am in his view.

**Comment by tom5454** [comment]
Do you have `minecraft comes alive`?

**Comment by SyntaxError73** [comment]
Yes i do. Is there a workaround for if i have both mods installed?

**Comment by tom5454** [comment]
edit your mca config and add cpm to `playerRendererBlacklist`. `"cpm": "all"`

**Comment by SyntaxError73** [comment]
Thank you very much
